tck/pom.xml

```diff
@@ -70,19 +71,19 @@
             <artifactId>microprofile-reactive-streams-operators-api</artifactId>
             <scope>provided</scope>
         </dependency>
-        
+
```

Comment:
Remove the newlines added

Reply:
Its just removed trailing whitespace and I am literally unable to revert it https://youtrack.jetbrains.com/issue/IDEA-244932

Reply:
Fixed